projects/resources-and-entities.md

```diff
@@ -0,0 +1,122 @@
+
+>For example, a process producing telemetry that is running in a container on Kubernetes has a Pod name, it is in a namespace and possibly is part of a Deployment which also has a name. All three of these attributes can be included in the Resource.
+
+The problem with such usage is that by looking at the Resource attributes it is impossible to tell which of the represented entities is *the entity*, i.e. the entity which produced the telemetry.
+
+### Problem 2: Lack of Precise Identity
+
+The Resource is one set of attributes, which contains all attributes of all entities that the Resource represents. It is impossible to tell which of these attributes identify the entity (or entities) and which are non-identifying, i.e. purely descriptive.
+
+This lack of precise identity makes it difficult or impossible to identify the same entities reported in different Resources. 
+
+### Problem 3: Lack of Mutable Attributes
+
+Resource is defined to be immutable in the OpenTelemetry SDK. This does not align well with the fact that non-identifying attributes of entities may change over time. For example OpenTelemetry Collector collects data about Pods and adds Pod labels as Resource attributes. Pod labels are mutable in Kubernetes and can change over time, while the Pod's identity remains immutable. Here is another example where [mutable Service attributes](https://github.com/open-telemetry/opentelemetry-specification/issues/3401#issuecomment-1511770735) are desirable.
+
+With the current definition of the Resource we are forced to either leave out any attributes that may ever change over time or violate the spec definition.
+
+Additionally, OpenTelemetry currently lacks the ability to provide resource attributes that require some kind of delayed lookup that may fail (see this issue). This required, e.g. passing environment variables for k8s container name and various downward-api values for an OpenTelemetry SDK to appropriately report this resource.
+
+In reality OpenTelemetry SDKs can also easily violate the definition as soon as we consider mutability from recipients perspective. SDKs only guarantee immutability during a single process session. As soon as the process is restarted and the SDK is newly initialized there is no guarantee that the Resource will have the same set of attributes (e.g. because process.id can be one of the Resource attributes).
+It is clear that the strictly "immutable" definition of the Resource is not sufficient for what we are trying to model.
+
+### Problem 4: Metric Cardinality Problem
+
+Today every attribute in an OpenTelemetry Resource, according to the metric data model, is used to determine the identity of metric.  Given known issues in metric time-series database implementation around cardinality, this can cause major issues if Resources are allowed to leverage high cardinality attributes. 
```

Comment:
```suggestion
Today every attribute in an OpenTelemetry Resource, according to the metric data model, is used to determine the identity of a metric.  Given known issues in metric time-series database implementations around cardinality, this can cause major issues if Resources are allowed to leverage high cardinality attributes. 
```